src/TTextCodec.cpp

```diff
@@ -316,6 +382,51 @@ QString TTextCodec_869::convertToUnicode(const char *in, int length, ConverterSt
     return result;
 }
 
+// This one does have some invalid codes in the table so we need to also
+// need to spot any incoming (char)in values that correspond to the invalid
+// (replacement) codepoint:
+QString TTextCodec_medievia::convertToUnicode(const char *in, int length, ConverterState *state) const
+{
+    QString result;
+    bool headerDone = false;
```

Comment:
What do you think about calling it `parsedHeader`? It would be easier to understand it later down - `if parsed header then...`

Reply:
The documentation for `ConverterState` is somewhat opaque - IIRC I had to grok Qt's own source code to work out how to use it - the only details they seemed to have published is: https://doc.qt.io/archives/qt-5.12/qtextcodec.html#ConversionFlag-enum ...

Also the flag is about whether a header (actually a BOM - Byte Ordering Mark) has been inserted - not whether it has been "parsed" per se...